src/pages/lessons/[slug].tsx

```diff
@@ -146,6 +146,8 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
   const [watchCount, setWatchCount] = React.useState<number>(0)
   const currentPlayerState = playerState.value as string
 
+  const actualPlayerRef = React.useRef<any>(null)
```

Comment:
the `playerRef` above is the wrapper and not the player player, so here we are...